app/assets/v2/js/grants/index.js

```diff
@@ -328,10 +342,12 @@ if (document.getElementById('grants-showcase')) {
         if (filters.type === 'collections') {
           this.collectionsPage = 1;
         }
-
         this.page = 1;
         this.setCurrentType(this.current_type);
         this.fetchGrants(this.page);
+        const regex_style = document.all_routing_policies.find(policy => new RegExp(policy.url_pattern).test(window.location.href))
+        toggleStyle(regex_style || current_style)
```

Comment:
instead of doing regex again why not the server give a json with the img url and categories/types to apply?

Reply:
That's quite hard to do, because it's impossible to be able to determine all possible category/types.